api/src/router/routes/quota.ts

```diff
@@ -30,5 +34,9 @@ router.get("/", asyncMiddleware(fetchQuota));
 // Utilizes raw data from DB
 // and transfroms to usable data for the front end
 router.get("/sizes", asyncMiddleware(fetchQuotaSizes));
+router.get(
+  "/get-all-available-quota-sizes",
```

Comment:
the route is a `GET` so `/get-all-available-quota-sizes` is not aweomse.

how about `GET /quota-sizes`?